### PR TITLE
Add suffixes to typos from "sa" to "se"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -45286,8 +45286,17 @@ sabatoshing->sabotaging
 sabatour->saboteur
 sacalar->scalar
 sacalars->scalars
+sacale->scale
+sacaled->scaled
+sacales->scales
+sacaling->scaling
 sacarin->saccharin
-sacle->scale
+saclar->scalar
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:45294]: Any typos of `sacral` and `secular`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -45545,6 +45573,10 @@ scetch->sketch
 scetched->sketched
 scetches->sketches
 scetching->sketching
+scetion->section
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:45576]: Perhaps `secretion`?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -46181,6 +46314,9 @@ sepearte->separate
 sepearted->separated
 sepeartely->separately
 sepeartes->separates
+sepearting->separating
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:46317]: Perhaps `departing` or `repeating`?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:45294]: I'll add `sacral`. I couldn't find `secular`.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:45576]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:46317]: I couldn't find these typos.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**